CribblyBackend/Services/TeamService.cs

```diff
@@ -0,0 +1,86 @@
+using System.Data;
+using System.Linq;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using CribblyBackend.Models;
+using Dapper;
+
+namespace CribblyBackend.Services
+{
+    public interface ITeamService
+    {
+        Task<Team> GetById(int Id);
+        void Update(Team Team);
+        Task Create(Team Team);
+        void Delete(Team Team);
+    }
+    public class TeamService : ITeamService
+    {
+        IDbConnection connection;
+        public TeamService(IDbConnection connection)
+        {
+            this.connection = connection;
+        }
+
+        public async Task Create(Team team)
+        {
+            await connection.ExecuteAsync(
+                @"INSERT INTO Teams(Name) VALUES (@Name)", 
+                new { Name = team.Name }
+            );
+            foreach (Player player in team.Players)
```

Comment:
We know there are at most 2 players because we're checking that in the controller, but it feels like we should check that at the service level so we don't accidentally somehow update a ton of players with a team ID.